llms.txt

```diff
@@ -0,0 +1,280 @@
+# Velib MCP Server - LLM Integration Guide
+
+## Overview
+This is a Model Context Protocol (MCP) server providing comprehensive access to Paris Velib bike-sharing data. The server offers real-time bike availability, station locations, and journey planning capabilities optimized for LLM consumption.
+
+## Quick Start
+To integrate with this MCP server:
+
+1. **Server URL**: `http://localhost:8080` (local development)
+2. **Protocol**: JSON-RPC 2.0 over HTTP/WebSocket
+3. **Documentation Endpoint**: Call `docs/schema` method or access `velib://docs/schema` resource
+
+## Key Features for LLMs
+
+### Real-time Data Access
+- **Live bike availability**: Updated every 60 seconds
+- **Station locations**: 1,400+ stations across Paris
+- **Journey planning**: Optimal pickup/dropoff recommendations
+- **Area statistics**: Aggregated metrics for geographic regions
+
+### Geographic Coverage
+- **Service Area**: Paris metropolitan area (50km radius from City Hall)
+- **Coordinates**: Latitude 48.7-49.0°N, Longitude 2.0-2.6°E
+- **Stations**: Complete coverage of Paris and surrounding communes
+
+### LLM-Optimized Features
+- **Self-documentation**: Comprehensive schema information available at runtime
+- **Clear error messages**: Actionable guidance for fixing issues
+- **Usage examples**: Detailed examples for every tool and endpoint
+- **Performance notes**: Response times and caching information
+- **Token efficiency**: Compact response formats available
+
+## Available Tools
+
+### 1. find_nearby_stations
+Find bike stations within walking distance of a location.
+- **Purpose**: Locate stations for pickup or dropoff
+- **Constraints**: 5km max radius, 100 station limit
+- **Typical use**: "Find stations with electric bikes near the Louvre"
+
+### 2. get_station_by_code
+Get detailed information about a specific station.
+- **Purpose**: Retrieve complete station details
+- **Input**: Station code (numeric string)
+- **Typical use**: "Get current status of station 32017"
+
+### 3. search_stations_by_name
+Search stations by name with fuzzy matching.
+- **Purpose**: Find stations by landmark or street name
+- **Features**: Fuzzy search, case-insensitive
+- **Typical use**: "Find stations near 'Eiffel Tower'"
```

Comment:
How can we help models demystify this searching mechanism? 

How well does this endpoint handle non-French fuzzy matching (here the example use is in English)?